SEDaily-IOS/AudioView.swift

```diff
@@ -107,6 +109,7 @@ class AudioView: UIView {
             guard currentSpeed != newValue else { return }
             self.playbackSpeedButton.setTitle(newValue.shortTitle, for: .normal)
             self.delegate?.audioRateChanged(newRate: newValue.rawValue)
+            UserDefaults.standard.set(newValue.rawValue, forKey: AudioView.userSettingPlaybackSpeedKey)
```

Comment:
Maybe we should be doing UserDefaults stuff in the manager instead, in the interest of keeping the view quite minimal and dumb? There's already a delegate method (audioRateChanged) in the manager so that seems like a good place to write to UserDefaults. The manager can then keep the key itself too instead of having to refer to the AudioView class to get it.

Reply:
Good catch! Thanks, will move it to audioRateChanged.